.ruby-gemset

```diff
@@ -0,0 +1 @@
+sms2fa-sinatra
```

Comment:
@ronualdo In order to keep this files here, I think we should give more context about the use of this in the readme.